cmd/deploy.go

```diff
@@ -12,13 +12,29 @@ import (
 )
 
 var (
+	deployExample = `
+Deployment you would like to deploy to Airflow cluster:
+
+  $ astro deploy <deployment name>
+
+Deploy deployment using suggestion:
+
+  $ astro deploy
+
+  Select which airflow deployment you want to deploy to:
+   #     LABEL                   DEPLOYMENT NAME            WORKSPACE     DEPLOYMENT ID
+   1     new-deployment-name     physical-diameter-1566     w1            ck1ryz2jd00430f50a5dmu7g9
```

Comment:
@andriisoldatenko it feels weird having dummy values in the example here. It's a small thing but I wouldn't want to confuse folks with fake data in here. Maybe we drop the example output here and just indicate that you will be presented a menu if you do not specify a deployment name.